src/main/frontend/handler/common.cljs

```diff
@@ -241,3 +233,14 @@
       (d/set-style! context-menu
                     :left (str client-x "px")
                     :top (str (+ scroll-y client-y) "px")))))
+
+(defn parse-config
+  "Parse configuration from file `content` such as from config.edn."
+  [content]
+  (try
+    (rewrite/parse-string content)
+    (catch js/Error e
```

Comment:
If we want to catch all exceptions I'd encourage using `(catch :default e` . More about `:default` at https://clojure.atlassian.net/browse/CLJ-1293. This may be why the example I QAed didn't work. I see a number of other catch's that use `js/Error`. I could make a card for converting those over as well if you'd like

Reply:
A card to convert all of them will be great!